### PR TITLE
added documentation + fixed ipynb files

### PR DESCRIPTION
**Pull request by dhrubo-os**
Signed-off-by: Dhrubo Saha <dhrubo@amazon.com>

### Description
[Describe what this change achieves]
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: docs/source/development/contributing.rst [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+eland <https://github.com/elastic/eland>`__ - your issue may already
+have been fixed. If not, search our `issues
+list <https://github.com/elastic/eland/issues>`__ on GitHub in case a
+similar issue has already been opened.
+
+It is very helpful if you can prepare a reproduction of the bug. In
+other words, provide a small test case which we can run to confirm your
+bug. It makes it easier to find the problem and to fix it. Test cases
+should be provided as python scripts, ideally with some details of your
+Elasticsearch environment and index mappings, and (where appropriate) a
```
Comment (ylwu-amzn) [inline docs/source/development/contributing.rst:25]: change to opensearch?

Comment (dhrubo-os) [reply docs/source/development/contributing.rst:25]: I'll delete this file in the next PR (mistakenly uploaded). I'm referring to our [Contributing.md](https://github.com/opensearch-project/opensearch-py-ml/blob/main/CONTRIBUTING.md)

File: docs/source/reference/supported_apis.rst [hunk 2]
```diff
@@ -0,0 +1,1207 @@
...
+    processing. Functions like ``DataFrame.iloc[i]`` or ``DataFrame.transpose()``
+    likely won't be implementable in Opensearch-py-ml due to this constraint.
+
+This list should be automatically generated with ``utils/generate-supported-apis.py``
+script instead of being modified manually.
+
++---------------------------------------+------------+
+| Method or Property                    | Supported? |
++=======================================+============+
+| ``ed.DataFrame.abs()``                | No         |
```
Comment (ylwu-amzn) [inline docs/source/reference/supported_apis.rst:34]: What does `ed` mean?

Comment (dhrubo-os) [reply docs/source/reference/supported_apis.rst]: Good Question. `ed` is used as short form of `eland`. I was looking at that file. This file is auto generated. I pushed a commit to delete this file and later I will fork the code to generate this rst by itself.

Comment (ylwu-amzn) [reply docs/source/reference/supported_apis.rst]: Cool, actually I see `ed` in other files too.

Comment (dhrubo-os) [reply docs/source/reference/supported_apis.rst]: Yeah, those are in the code files. To reflect/change the variable name, requires another larger PR, which I'm planning to do after this PR.

**Review by ylwu-amzn — APPROVED** [review]

**Pull request merged**